### ISSUE TITLE
Compilation error on Android 

### ISSUE THREAD

**Issue by ezraroda** [AzureAD/azure-activedirectory-library-for-cordova#62]
Getting the following error 👎 

```
compileReleaseJavaWithJavac/...
/platforms/**android/src/com/microsoft/aad/adal/CordovaAdalPlugin.java:162**: error: incompatible types: List<TokenCacheItem> cannot be converted to ArrayList<TokenCacheItem>
                ArrayList<TokenCacheItem> tokensForUserId = ((ITokenStoreQuery)cache).getTokensForUser(userId);
```
                                                                                                      ^

Running with the following configuration:

Cordova CLI: 6.1.1
Gulp version:  CLI version 3.9.1
Gulp local:   Local version 3.9.1
Ionic Framework Version: 2.0.0-beta.6
Ionic CLI Version: 2.0.0-beta.25
Ionic App Lib Version: 2.0.0-beta.15
ios-deploy version: 1.8.3 
ios-sim version: 5.0.4 
OS: Mac OS X El Capitan

java version "1.8.0_45"
Java(TM) SE Runtime Environment (build 1.8.0_45-b14)
Java HotSpot(TM) 64-Bit Server VM (build 25.45-b02, mixed mode)

cordova platform info
Installed platforms:
  **android 5.1.1**

it probably require explicit cast:
`ArrayList<TokenCacheItem> tokensForUserId = (ArrayList<TokenCacheItem>)((ITokenStoreQuery)cache).getTokensForUser(userId);`

**Comment by rwdixon69** [comment]
I'm seeing this identical error, and am unable to compile for the Android platform.

**Comment by nikhilkh** [comment]
@riknoll to help investigate

**Comment by yonatan-onoapps** [comment]
explicit cast fixed it

**Comment by nikhilkh** [comment]
This has been fixed in 0.7.1 version now published on npm

**Comment by rwdixon69** [comment]
I can confirm that the plugin will now compile.  Thank you!
